ci-operator/step-registry/openshift-pipelines/scaling-pipelines/openshift-pipelines-scaling-pipelines-commands.sh

```
@@ -65,11 +65,45 @@ if [ "$JOB_TYPE" == "presubmit" ] && [[ "$JOB_NAME" != rehearse-* ]]; then
+    # Multi-scenario mode: run multiple parameter combinations on the same cluster.
+    # Format: "total/concurrent/namespace/steps" (space-separated list)
+
+    cleanup_namespaces() {
+        for ns_idx in $(seq 1 "${TEST_NAMESPACE}"); do
+            ns_tag=$([ "$TEST_NAMESPACE" -eq 1 ] && echo "" || echo "$ns_idx")
+            oc delete --cascade=foreground --timeout=30m namespace "benchmark${ns_tag}" 2>/dev/null || true
+        done
+    }
+
+    for scenario in $TEST_SCENARIOS; do
+        export TEST_TOTAL="$(echo "$scenario" | cut -d "/" -f 1)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)